src/ray/stats/stats.h

```diff
@@ -115,6 +112,18 @@ static inline void Init(
   StatsConfig::instance().SetIsInitialized(true);
 }
 
+static inline void InitOpenTelemetryMetricAgent(const int metrics_agent_port) {
+  if (!RayConfig::instance().experimental_enable_open_telemetry_on_core()) {
+    return;
+  }
+  OpenTelemetryMetricRecorder::GetInstance().RegisterGrpcExporter(
+      std::string("127.0.0.1:") + std::to_string(metrics_agent_port),
+      std::chrono::milliseconds(
+          absl::ToInt64Milliseconds(StatsConfig::instance().GetReportInterval())),
+      std::chrono::milliseconds(
+          absl::ToInt64Milliseconds(StatsConfig::instance().GetHarvestInterval())));
```

Comment:
should add inline comments for what these params are, like: `/*foobar_timeout_ms=*/...`